controllers/user_controller.py

```
@@ -0,0 +1,17 @@
+from fastapi import APIRouter, Depends
+from sqlalchemy.orm import Session
+from database import get_db
+from models import User
+from logger import get_logger
+
+router = APIRouter(prefix="/users", tags=["users"])
+logger = get_logger("user_controller")
+
+@router.get("/")
+def read_users(db: Session = Depends(get_db)):
```

Ruff (0.8.2)

Do not perform function call `Depends` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)